ui_framework/components/tool_bar/tool_bar_search_box.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, mount } from 'enzyme';
+import sinon from 'sinon';
+import { commonHtmlProps } from '../../test/common_html_props';
+
+import {
+  KuiToolBarSearchBox,
+} from './tool_bar_search_box';
+
+describe('KuiToolBarSearchBox', () => {
```

Comment:
>  they should be quick and easy to write

💯 ! Let me rephrase my comment in the form of a problem instead of a solution. Then you can tell me if you can relate or if I'm possibly alone in this.

I have a hard time reading tests like https://github.com/elastic/cloud/blob/master/cloud-ui/public/components/Alert/__tests__/Alert.test.js. I think it's hard for me to quickly grok this test because I have to read each test's name to search for patterns in the tests, and sometimes the way the test names are worded make it difficult for me to compare and contrast because there's so much repetition in each test name.

So this is why I'm reaching for describe blocks as a solution. They're just a form of documentation, like comments, except they also create a visual relationship. They create context, both for their children and in relation to one another, which is one way I've solved the problem above (I'm open to alternatives though). They also reduce the repetition in each test name, so the test names are easier to compare.

So what do you think? Any ideas on how to solve my problem in a way that doesn't create unreasonable friction for us?